Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM appwrite/appwrite:dev
+
+LABEL maintainer="team@appwrite.io"
```

Comment:
Would be a nice idea to add the `_APP_EDITION` env var and overwrite the default `community` value with `community-lite` we can use it later to distinguish usage stats.

Reply:
In addition, maybe we can add some of the .env file settings directly in this Dockerfile

Reply:
done!